cli/src/diff_util.rs

```diff
@@ -387,13 +387,20 @@ fn show_color_words_diff_line(
                 formatter.write_all(data)?;
             }
             DiffHunk::Different(data) => {
-                let before = data[0];
-                let after = data[1];
-                if !before.is_empty() {
-                    formatter.with_label("removed", |formatter| formatter.write_all(before))?;
-                }
-                if !after.is_empty() {
-                    formatter.with_label("added", |formatter| formatter.write_all(after))?;
+                let labels = ["removed", "added"];
+                for (index, label) in labels.iter().enumerate() {
```

Comment:
nit: you can use `.iter().zip()` instead

Reply:
Done.